spec/index.html

```diff
@@ -657,9 +657,51 @@ <h3>Result</h3>
   </aside>
 </emu-clause>
 
+<emu-clause id="emu-grammar">
+  <h1>emu-grammar</h1>
+  <p>Text inside emu-grammar elements is parsed using <a href="https://github.com/rbuckton/grammarkdown">Grammarkdown</a>. The syntax is essentially identical to the notational conventions in ECMAScript (minus formatting). See the <a href="https://github.com/rbuckton/grammarkdown">Grammarkdown readme</a>.</p>
+  <p>Grammar will be displayed as an inline element flowing with text unless it is the immediate child of an emu-clause-like element.</p>
+
+  <h2>Attributes</h2>
+  <p><b>collapsed:</b> If present, productions are displayed in collapsed format with LHS and RHS on the same line.</p>
+  <p><b>example:</b> If present, the element is an example.</p>
+  <p><b>primary:</b> Optional: Deprecated in favor of type="definition".</p>
+  <p><b>type:</b> Optional: Disposition of the grammar. If absent, it is considered to reference productions defined elsewhere.</p>
+  <ul>
+    <li>"definition": The grammar is an authoritative source for productions which should be the target of references.</li>
+    <li>"example": Deprecated in favor of the <b>example</b> attribute.</li>
+  </ul>
+
+  <h2>Example</h2>
```

Comment:
Should the emu-grammar examples be extended to include the more complex grammar currently at https://tc39.es/ecmarkup/#grammar , or is that best relegated to Grammarkdown anyway?

Reply:
I'm fine with just having the simpler examples and a link.